src/pages/guides/flutter.mdx

```diff
@@ -0,0 +1,307 @@
+
+## Prerequisites
+
+---
+
+<Web3AuthPrerequisites />
+
+- Android API version 21 or newer.
+- `compileSdkVersion` 32 or higher
+
+## Setup
+
+---
+
+### Setup your Web3Auth Dashboard
+
+<SetupWeb3AuthDashboard />
+
+- Add `{YOUR_APP_PACKAGE_NAME}://auth` in the **Whitelist URL** field of the Web3Auth Dashboard.
+
+### Configure Deep Link
+
+Open your app's `AndroidManifest.xml` file and add the following deep link intent filter to your sign-in activity:
+
+```xml
+<intent-filter>
+  <action android:name="android.intent.action.VIEW" />
+
+  <category android:name="android.intent.category.DEFAULT" />
+  <category android:name="android.intent.category.BROWSABLE" />
+
+  <!-- Accept URIs: {YOUR_APP_PACKAGE_NAME}://* -->
+  <data android:scheme="{YOUR_APP_PACKAGE_NAME}" />
+</intent-filter>
+```
+
+## Using the Web3Auth SDK
+
+---
+
+To use the Web3Auth SDK, you need to add the dependency of the respective platform SDK of Web3Auth to your project. To know more about the available
+SDKs, please have a look at this [documentation page](/docs/sdk).
+
+For this guide here, we will be talking through the [Web3Auth Flutter SDK](/sdk/flutter).
+
+### Installation
+
+#### Add Web3Auth to Gradle
+
+In your project-level `settings.gradle` or `build.gradle` file, add JitPack repository:
+
+```groovy
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        // highlight-next-line
+        maven { url "https://jitpack.io" } // <-- Add this line
+    }
+}
+```
+
+Then, in your app-level `build.gradle` dependencies section, add the following:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'org.torusresearch:web3auth-android-sdk:2.5.0'
+}
+```
+
+#### Permissions
+
+Open your app's `AndroidManifest.xml` file and add the following permission:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+> Note: The `<uses-permission>` element must be a direct child of the `<manifest>` root element
+
+### Initialization
+
+After Installation, the next step to use Web3Auth is to Initialize the SDK. This can be achieved by using the `init()` function.
+
+Please note that these are the most critical steps where you need to pass on different parameters according to the preference of your project.
+Additionally, Whitelabeling and Custom Authentication have to be configured within this step, if you wish to customise your Web3Auth Instance.
+
+#### Create Web3Auth Instance
+
+In your `main.dart` file, initialise the `Web3AuthFlutter` plugin at the very beginning such as in the overriden `initState` function
+
+```dart
+import 'dart:collection';
+import 'dart:io';
+import 'dart:async';
+
+import 'package:web3auth_flutter/web3auth_flutter.dart';
+import 'package:web3auth_flutter/enums.dart';
+import 'package:web3auth_flutter/input.dart';
+import 'package:web3auth_flutter/output.dart';
```

Comment:
why do we need to import all of these packages?

Reply:
It is from the examples folder of Flutter SDK repo.